content/docs/command-reference/update.md

```diff
@@ -49,6 +49,17 @@ $ dvc update --rev master
   directory and its subdirectories for import stage `.dvc` files to inspect. If
   there are no directories among the targets, this option is ignored.
 
+- `--to-remote` - sync the changes directly on the remote storage, rather than
```

Comment:
1. it's not very descriptive. Can we at least reference `--to-remote` for the `dvc add`/`dvc import-url`. That this option corresponds to those, etc.
2. does it work for the regular `dvc import`? what happens if I run it?

Reply:
> does it work for the regular dvc import? what happens if I run it?

iterative/dvc#5475 should forbid using with import (currently you get an irrelevant error).

Reply:
> it's not very descriptive. Can we at least reference --to-remote for the dvc add/dvc import-url. That this option corresponds to those, etc.

@jorgeorpinel 's suggestion seems fixing it, I can commit it if there is nothing blocks?

Reply:
resolving this one, let's keep discussion in the @jorgeorpinel suggestion ...